doc_src/functional_hazard_analysis_longline/fha_longline.tex

```diff
@@ -355,34 +358,847 @@ \subsection{Phase 1: Approach and Positioning}
+	     fhaMitigationKeyValue=
+	     	{\tabitem Abrasion-resistant coating of line and backups\\
+	     	\tabitem \textcolor{purple}{patient} and \textcolor{teal}{rescuer} wear an appropriate helmet\\
+	     	\tabitem Line construction with multiple backups made of core-sheath ropes\\
+	     	\tabitem Sufficient fall height under the highline for stretching by weight of two persons\\
+	     	\tabitem Terrestrial rescue equipment and an easily accessible highline stand\\
+	     	\tabitem Sufficient personnel with knowledge at the anchorage for a terrestrial rescue},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{red}{high}. The sheathing prevents friction damage to the line.
+	      	Damage caused by impact can be mitigated by the backups used.
+ 			Injury to people caused by the falling rope cannot be ruled out, but immediate, rapid rescue can be prepared.
+			The helmet offers head protection. A sufficient fall height can be planned for when setting up.}]
+
+%2.2
+\fhaTable[fhaSystemKeyValue=
+			{helicopter},
+		 fhaFunctionKeyValue=
+		 	{positioning},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.2},
+	     fhaCausesKeyValue=
+	     	{Tied-up helicopter steers the highline due to position fluctuations}, 
+	     fhaDescriptionKeyValue=
+	     	{During the positioning of the \textcolor{teal}{rescuer} to the \textcolor{purple}{patient} by the helicopter, relative movement with contact between the rope and the highline could occur for various reasons.
+	      	This could cause friction damage to the highline.
+	      	Furthermore, the contact between the line and the rope could affect the control behavior of the helicopter.
+	      	In the most serious case, the rope could have to be released (in this case see 2.1)}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Abrasion-resistant coating of line and backups\\
+	     	\tabitem Line construction with multiple backups made of core-sheath ropes\\},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{orange}{medium}, as the sheath prevents friction damage to the line. Damage caused by hitting can be mitigated by the backups used.
+ 			Dropping the rope does not cause the \textcolor{teal}{rescuer} to fall}]
+ 
+%2.3 
+\fhaTable[fhaSystemKeyValue=
+			{\textcolor{teal}{rescuer}},
+		 fhaFunctionKeyValue=
+		 	{positioning (transl.)},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.3},
+	     fhaCausesKeyValue=
+	     	{Flight \textcolor{teal}{rescuer} positions himself too far below \textcolor{purple}{patient} to attach \textcolor{purple}{patient} to rope.}, 
+	     fhaDescriptionKeyValue=
+	     	{Immediately after hooking in, the \textcolor{teal}{rescuer} can lower himself too far with the variable rope or hang below the \textcolor{purple}{patient} due to the helicopter losing altitude.
```

Comment:
inconsistent wording, sometimes "height", sometimes "altitude" is used
(not a big deal but consistency is nice..)

Reply:
done in https://github.com/JBludau/Highline-Rescue-Project/pull/10/commits/ac02c3b2bd7059da0bfd4b65be24c7f99dae9f61